daliuge-engine/docker/Dockerfile.devall

```diff
@@ -25,11 +25,11 @@ ENV PATH="$VIRTUAL_ENV/bin:$PATH"
 ENV DLG_ROOT="/tmp/dlg"
 
 RUN apt install -y git python3-dev
-#RUN pip install 'git+https://github.com/ICRAR/dlg-nifty-components.git'
-#RUN pip install 'git+https://github.com/ICRAR/dlg-casacore-components.git'
 
-RUN pip install --extra-index-url=https://artefact.skao.int/repository/pypi-internal/simple 'ska-sdp-cbf-emulator[plasma]>=1.6.11'
+RUN pip install git+https://gitlab.com/ska-telescope/sdp/ska-gridder-nifty-cuda.git
 RUN pip install dlg-nifty-components
+
+RUN pip install --extra-index-url=https://artefact.skao.int/repository/pypi-internal/simple 'ska-sdp-cbf-emulator[plasma]==1.6.11'
```

Comment:
Almost overlooked this, but this should be updated to the newer sdp-realtime repositories, I'll do so after releasing the new version dlg-casacore-components to pypi then version locking

Reply:
The reason some of the ska repositories are listed here is I can't to get a PyPi package to grab a dependency from the SKA package repo, it only works when the command `pip install -r requirements.txt` is run